igneous/tasks/mesh/multires.py

```diff
@@ -111,7 +116,9 @@ def retriangulate_mesh(mesh: trimesh.Trimesh, offset: "Vec", grid_size: "Vec", s
         new_mesh = trimesh.util.concatenate(new_mesh, mesh_z)
   return new_mesh
 
-def determine_mesh_shape_from_lods(lods: list[trimesh.Trimesh]):
+def determine_mesh_shape_from_lods(
+    lods: list[trimesh.Trimesh],
+  ) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
I recently learned that the right way to do this is:

`import numpy.typing as npt`

`tuple[npt.NDArray[np.float32], npt.NDArray[np.int_]]`

Reply:
Oh nice, thank you!